borg/archiver.py

```diff
@@ -2033,24 +2032,25 @@ def prerun_checks(self, logger):
 
     def _setup_implied_logging(self, args):
         """ turn on INFO level logging for args that imply that they will produce output """
-        # map of option name to name of child logger (which matches the option name)
+        # map of option name to name of logger for that option
         option_logger = {
-                'output_list': 'list',
-                'show_version': 'show-version',
-                'show_rc': 'show-rc',
-                'stats': 'stats',
+                'output_list': 'borg.output.list',
+                'show_version': 'borg.output.show-version',
```

Comment:
Python doesn't document any requirements, other than dots make hierarchy.  A glance at the implementation doesn't show any requirements either.

I did test, with this config: https://gist.github.com/edgewood/7e3bd5def843d50012c1b4cee5257303